Editor/Analytics/AmplitudeEditorLogger.cs

```diff
@@ -262,24 +274,25 @@ private void GenerateSessionId()
 
         private void ToggleAnalytics(bool allow)
         {
-            LogEvent(Constants.EventName.ALLOW_ANALYTICS, new Dictionary<string, object>
+            LogEvent(EventName.ALLOW_ANALYTICS, new Dictionary<string, object>
             {
-                { Constants.Properties.ALLOW, allow }
+                { Properties.ALLOW, allow }
             }, new Dictionary<string, object>
             {
-                { Constants.Properties.ENGINE_VERSION, appData.UnityVersion },
-                { Constants.Properties.RENDER_PIPELINE, appData.RenderPipeline },
-                { Constants.Properties.SUBDOMAIN, appData.PartnerName },
-                { Constants.Properties.APP_NAME, PlayerSettings.productName },
-                { Constants.Properties.SDK_TARGET, "Unity" },
-                { Constants.Properties.APP_IDENTIFIER, Application.identifier },
-                { Constants.Properties.ALLOW_ANALYTICS, allow }
+                { Properties.ENGINE_VERSION, appData.UnityVersion },
+                { Properties.RENDER_PIPELINE, appData.RenderPipeline },
+                { Properties.SUBDOMAIN, appData.PartnerName },
+                { Properties.APP_NAME, PlayerSettings.productName },
+                { Properties.SDK_TARGET, "Unity" },
+                { Properties.APP_IDENTIFIER, Application.identifier },
+                { Properties.ALLOW_ANALYTICS, allow }
             });
         }
-        
-        private void LogEvent(string eventName, Dictionary<string, object> eventProperties = null, Dictionary<string, object> userProperties = null)
+
+        private void LogEvent(string eventName, Dictionary<string, object> eventProperties = null, Dictionary<string, object> userProperties = null, bool force = false)
         {
-            if (!isEnabled) return;
+            if (!isEnabled && !force) return;
```

Comment:
We should not be doing this. If somebody have disabled analytics, we shouldn't send any events. 
@bfercher Please confirm whether this is correct.

Reply:
Yup, it's primary reason for existing is because you can't opt in to analytics before you install the package... so we have to skip the analytics check in this use-case if we want to capture the package install event information. We could do it a different way which doesn't introduce a 'force' boolean, but it would still be doing the same thing 😅

Reply:
Found a way to achieve the same thing whilst waiting for consent from the dev 🙏